src/client/src/containers/DependencyInfo/index.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import styles from "./styles.module.css";
+import * as getSvg from "../../utils/getSvgUrl";
+import classnames from "classnames";
+import { injectIntl, defineMessages } from "react-intl";
+import { WIZARD_CONTENT_INTERNAL_NAMES } from "../../utils/constants";
+import { AppState } from "../../reducers";
+import { IDependenciesInstalled } from "../../reducers/dependencyInfoReducers";
+
+interface IDependencyInfoProps {
+  frameworkName: string;
+  intl: any;
+  dependenciesStore: IDependenciesInstalled;
+}
+
+type Props = IDependencyInfoProps;
+
+const messages = defineMessages({
+  installed: {
+    id: "dependencyChecker.installed",
+    defaultMessage: " detected!"
+  },
+  notInstalled: {
+    id: "dependencyChecker.notInstalled",
+    defaultMessage: " not detected. Click to install."
+  },
+  iconAltMessage: {
+    id: "dependencyChecker.iconAltMessage",
+    defaultMessage: "Icon for dependency checker"
+  }
+});
+
+interface IDependency {
+  [key: string]: string;
+}
+
+interface IDependencies {
+  [key: string]: IDependency;
+}
+
+const dependencies: IDependencies = {
+  NodeJS: {
```

Comment:
This is great! Do you think it would be possible to move this metadata to `backendframeworks.json` on the `templates/_catalog` folder?

Reply:
Yea, I was considering this at first but put it in here because it would be simpler for now

Reply:
I don't think I will be moving it to `backendframeworks.json` in this PR, but i agree it would be easier to update in the future

Do you think maybe it would make sense to have a separate `dependency.json` in the catalog? @jcoc611-microsoft

Reply:
I think for this to work you would have to also change CoreTS to pick it up? Not sure, but I'm alright with the way you have it for now.